src/components/task/existing-task/edit-task/index.module.scss

```diff
@@ -0,0 +1,34 @@
+@import "styles/base.scss";
+
+.textArea {
+  outline: 3px solid $violet-color1;
+  border: none;
+  padding: 0;
+  border-radius: 0.313rem;
+  width: calc(100% - 1rem);
+  padding: 0.25rem 0.5rem;
+  color: $grey-color1;
+  line-height: 1.172rem;
+  font-size: 1rem;
```

Comment:
Just wanted to share a practice that we follow in WellTravel. We do order the css properties alphabetically (a-z). 💡